[2024-01-17T19:06:45.766-0300] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cities_coleta_teste.df scheduled__2024-01-17T22:05:32.598644+00:00 [queued]>
[2024-01-17T19:06:45.768-0300] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cities_coleta_teste.df scheduled__2024-01-17T22:05:32.598644+00:00 [queued]>
[2024-01-17T19:06:45.768-0300] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-01-17T19:06:45.778-0300] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): df> on 2024-01-17 22:05:32.598644+00:00
[2024-01-17T19:06:45.782-0300] {standard_task_runner.py:57} INFO - Started process 82434 to run task
[2024-01-17T19:06:45.784-0300] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'cities_coleta_teste', 'df', 'scheduled__2024-01-17T22:05:32.598644+00:00', '--job-id', '2441', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp4ep69xou']
[2024-01-17T19:06:45.784-0300] {standard_task_runner.py:85} INFO - Job 2441: Subtask df
[2024-01-17T19:06:45.801-0300] {task_command.py:416} INFO - Running <TaskInstance: cities_coleta_teste.df scheduled__2024-01-17T22:05:32.598644+00:00 [running]> on host desktop
[2024-01-17T19:06:45.828-0300] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='datasetpibic@gmail.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='cities_coleta_teste' AIRFLOW_CTX_TASK_ID='df' AIRFLOW_CTX_EXECUTION_DATE='2024-01-17T22:05:32.598644+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-17T22:05:32.598644+00:00'
[2024-01-17T19:06:46.419-0300] {logging_mixin.py:154} INFO - Called | 1705529206.4170513 | df
[2024-01-17T19:06:46.427-0300] {logging_mixin.py:154} INFO - SAVING FILE df
[2024-01-17T19:06:46.427-0300] {python.py:194} INFO - Done. Returned value was: None
[2024-01-17T19:06:46.430-0300] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=cities_coleta_teste, task_id=df, execution_date=20240117T220532, start_date=20240117T220645, end_date=20240117T220646
[2024-01-17T19:06:46.476-0300] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-01-17T19:06:46.482-0300] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
